sass imports do not work in build-mode

If I import a css or sass file like so:
```
@import "~normalize.css/normalize";
```

... it works great with `npm run start`, however if I use `npm run build` it doesn't work and says:

```
ERROR in ./src/index.scss
Module build failed: ModuleNotFoundError: Module not found: Error: Cannot resolve module '~normalize.css/normalize'
```

--- a/webpack.production.config.js
+++ b/webpack.production.config.js
@@ -17,7 +17,7 @@ loaders.push({
 loaders.push({
 	test: /[\/\\]src[\/\\].*\.scss/,
 	exclude: /(node_modules|bower_components|public)/,
-	loader: ExtractTextPlugin.extract('style', 'css?modules&importLoaders=1&localIdentName=[name]__[local]___[hash:base64:5]', 'sass')
+	loader: ExtractTextPlugin.extract('style', 'css!sass?modules&importLoaders=1&localIdentName=[name]__[local]___[hash:base64:5]')
 });
 // global css files
 loaders.push({

